blocks/page_list/templates/itinerary/ItineraryConstants.js

```diff
@@ -3,4 +3,7 @@ export default {
 	ADD_WALK: 'ADD_WALK',
 	UPDATE_TITLE: 'UPDATE_TITLE',
 	UPDATE_DESCRIPTION: 'UPDATE_DESCRIPTION',
-};
+	VIEW_LIST: 'VIEW_LIST',
+	CREATE_LIST: 'CREATE_LIST',
+	WALK_SELECTED: 'WALK_SELECTED',
```

Comment:
This is a really verbose way to do it, and you're also defining one thing called "ItineraryConstants", which doesn't really describe what these constants are. It would make more sense to have an export for Actions, and then mirror them from an array, e.g.

```
const Actions = [
  'ADD_WALK',
  'UPDATE_TITLE',
  'VIEW_LIST'
].reduce((p, v) => {p[v] = v; return p}, {});
export Actions;
```